package.json

```diff
@@ -1,14 +1,20 @@
 {
-  "name": "cast-monorepo",
+  "name": "cast",
   "version": "0.1.0",
-  "description": "A monorepo for the Cast app",
-  "private": true,
```

Comment:
[issue](https://conventionalcomments.org/#labels): this should probably still remain since we don't want to publish our app to npm.

Reply:
Returned.